cirq-web/cirq_ts/README.md

```diff
@@ -0,0 +1,96 @@
+ - Any `assets/` directory with information necessary for the visualization (fonts, images, etc.). In the case of the Bloch Sphere, we can see a `fonts/` subdirectory which holds necessary font data, within the `assets/` directory, but for instances where there isn't a lot of extra information necessary subdirectories may not be needed.
+ - A class within the visualization's "root" folder which brings the individual components of the visualization together. In the case of the Bloch Sphere, this is `BlochSphere.class.ts`.
+ - A `main.ts` consisting of functions which will be called from the bundled library. These function should handle:
+    - Receiving any input data which could affect the visualization.
+    - Sending final visualization output to the development environment, notebook, or HTML files. 
+    - Combining aspects of the visualization that need to be added separately.
+
+This `main.ts` file will also need to be added as an entry point in the `webpack.config.js` file in order for your visualization to be bundled accordingly.
+```javascript
+module.exports = {
+  entry: {
+    bloch_sphere: './src/bloch_sphere/main.ts',
+    ...
+    YOUR_VIZ_NAME: './src/YOUR_VIZ_NAME/main.ts',
+  },
+  ...
+};
+```
+You can learn more about Webpack entry points here: [Webpack Entry Points](https://webpack.js.org/concepts/entry-points/)
+
+### Creating visualization bundle files
+
+Following this structure, you will be able to bundle your visualization by running the command `npx webpack --mode production`. This will build the bundled Javascript file(s) within the `dist/` directory, where you can access and reference them in HTML.
+
+### Developing visualizations
+
+There are two main ways to develop visualizations you are creating in Cirq. The first, and recommended way, is to spin up a Webpack development server and view your visualizations in the browser. You can also develop using Jupyter notebook if you want to easily test integration with Python code. 
+
+#### Hot reloading development server (Recommended)
+Using `webpack-dev-server`, we are able to develop and test visualizations in the browser and have changes update as we're writing the code. You can start this server by running `npm run start` in this directory, and view your work on the port specified by Webpack. This method also requires an `index.html` file placed within the `dist` folder. You can also manually determine where your index file is served from by modifying where `webpack-dev-server` searches for files in the `webpack.config.js` file:
+```javascript
+module.exports = {
+  ...
+  devServer: {
+    static: path.join(__dirname, 'dist'),
+    public: 'localhost:8080',
+  },
+  ...
+};
+```
+Note that the bundled files that `webpack-dev-server` creates live in memory, so you won't be able to see them. However, you can access them like so:
+```html
+<script src="YOUR_VIZ_NAME.bundle.js"></script>
```

Comment:
```suggestion
<script src="/YOUR_VIZ_NAME.bundle.js"></script>
```

Right? It is served on the root of the context. 
We can also mention in addition that from the browser `http://localhost:8080/YOUR_VIZ_NAME.bundle.js` is a way to inspect the bundled js.